README.md

```diff
@@ -14,6 +14,8 @@ The main differences between this provisioner and the standard hostpath provisio
 3. Or if you do not want to specify the node on the claim, you can specify `volumeBindingMode: WaitForFirstConsumer` in the storage class. Then the PV will be created only when the first Pod using this PVC is scheduled. The PV will be created on the node that the Pod is scheduled on.
 Still, the annotation `kubevirt.io/provisionOnNode` can be used in this mode, though it will not wait for the first consumer.
 
+_There is on caveat with using the annotation, if you have a pod that uses multiple PVCs and you annotate one, you should annotate all of them otherwise there can be a race condition where the Kubernetes scheduler can decide to schedule the pod on a node you did not specify in your annotation._
```

Comment:
...There is one caveat...

Reply:
I think this could be clarified...

In cases where multiple PVCs are to be used with a Pod it is not recommended to mix the WaitForFirstConsumer binding mode with the provisionOnNode annotation.  All of a Pod's PVCs should carry the annotation or none should.  Mixing modes can result in PVCs being allocated from different nodes leaving your Pod unschedulable.